pennylane/kernels/cost_functions.py

```diff
@@ -105,33 +117,9 @@ def kernel_polarization(X, Y, kernel, assume_normalized_kernel=False, rescale_cl
 
     T = np.outer(_Y, _Y)
 
-    return _matrix_inner_product(K, T)
-
-
-def kernel_target_alignment(
-    X, Y, kernel, assume_normalized_kernel=False, rescale_class_labels=True
-):
-    """Kernel target alignment of a given kernel function.
-
-    Args:
-        X (list[datapoint]): List of datapoints
-        Y (list[float]): List of class labels of datapoints, assumed to be either -1 or 1.
-        kernel ((datapoint, datapoint) -> float): Kernel function that maps datapoints to kernel value.
-        assume_normalized_kernel (bool, optional): Assume that the kernel is normalized, i.e.
-            that when both arguments are the same datapoint the kernel evaluates to 1. Defaults to False.
+    return matrix_inner_product(K, T, normalize=normalize)
 
-    Returns:
-        float: The kernel target alignment.
-    """
-    K = square_kernel_matrix(X, kernel, assume_normalized_kernel=assume_normalized_kernel)
-
-    if rescale_class_labels:
-        nplus = np.count_nonzero(np.array(Y) == 1)
-        nminus = len(Y) - nplus
-        _Y = np.array([y / nplus if y == 1 else y / nminus for y in Y])
-    else:
-        _Y = np.array(Y)
-
-    T = np.outer(_Y, _Y)
 
-    return _matrix_inner_product(K, T) / (np.linalg.norm(K, "fro") * np.linalg.norm(T, "fro"))
+kernel_target_alignment = lambda *args, **kwargs: kernel_polarization(
```

Comment:
I would write this out and not define it using a lambda, I think if you do it like that you also don't get a proper docstring